src/drive/web/modules/drive/Toolbar/components/CreateNoteItem.jsx

```diff
@@ -8,21 +8,37 @@ import {
   useAppLinkWithStoreFallback,
   useCapabilities
 } from 'cozy-client'
-import { generateUniversalLink } from 'cozy-ui/transpiled/react/AppLinker/native'
+import {
+  generateUniversalLink,
+  generateWebLink
+} from 'cozy-ui/transpiled/react/AppLinker/native'
+import { ActionMenuItem } from 'cozy-ui/transpiled/react/ActionMenu'
+import Icon from 'cozy-ui/transpiled/react/Icon'
 
 import toolbarContainer from 'drive/web/modules/drive/Toolbar/toolbar'
-import styles from 'drive/styles/toolbar.styl'
 
 const CreateNoteItem = ({ client, t, displayedFolder }) => {
-  const { fetchStatus, url, isInstalled } = useAppLinkWithStoreFallback(
-    'notes',
-    client
-  )
   const capabilities = useCapabilities(client)
   const isFlatDomain = get(
     capabilities,
     'capabilities.data.attributes.flat_subdomains'
   )
+
+  let notesAppUrl = ''
+  const { fetchStatus, url, isInstalled } = useAppLinkWithStoreFallback(
+    'notes',
+    client
+  )
+  if (fetchStatus === 'loaded') {
+    notesAppUrl = url
+  } else if (fetchStatus === 'errored') {
```

Comment:
I don't know if the commit message is explicit enough, but on the public page we don't have permissions to query the list of apps, and this is one of the only places where we really need it. I think the fallback to a generated link is ok, the only issue is if the notes app is not installed, then we won't have a link to the store.

Reply:
> What happens if click on the button with no network?

It will try to open notes :/. 

> Can't we do a check if we're in a public context or not?

Yep, we can! The offline behavior would be the same though. But it's probably a better solution.

> (and no the commit was not self explaining, maybe you can add your comment to it ;))

👍

Reply:
cc @poupotte pour info ;)